1-exercises/D-object-methods/exercise.js

```diff
@@ -9,9 +9,13 @@
 
 let student = {
   // write code here
+  getName: function getName(name){
+    return name
+  }
 }
 
-student.getName("Daniel");
+ let studentName = student.getName("Daniel");
```

Comment:
It doesn't affect the code working, but we should only use `let` to define a variable if we are going to change it later on. In other cases, we should use `const`. This makes our code "safer" as we will only be able to reassign variables that we define as let, and expect to be changed. (Note: there are other subtle differences with let/const related to scope, but I wouldn't advise worrying about it at this stage).